corehq/apps/app_manager/views/forms.py

```diff
@@ -546,12 +547,19 @@ def get_form_view_context_and_template(request, domain, form, langs, messages=me
         form.get_unique_id()
         app.save()
 
+    allow_usercase = (domain_has_privilege(request.domain, privileges.USER_CASE)
+                      and not toggles.USER_TESTING_SIMPLIFY.enabled(request.domain))
+    valid_index_names = DEFAULT_CASE_INDEX_IDENTIFIERS.values()
+    if allow_usercase and USERCASE_PREFIX[-1] == "/":
```

Comment:
I'm curious why this check is here? are you trying to future-proof it against someone removing the trailing `/`? if so the check seems like it might go around the truncation of the prefix, not the addition of the prefix to `valid_index_names`